example/cumulus-tf/ingest_and_publish_granule_workflow.asl.json

```diff
@@ -237,6 +237,34 @@
         }
       ]
     },
+    "UpdateGranulesMetadataStep": {
+      "Parameters": {
+        "cma": {
+          "event.$": "$",
+          "task_config": {
+            "bucket": "{$.meta.buckets.internal.name}",
+            "buckets": "{$.meta.buckets}",
+            "distribution_endpoint": "{$.meta.distribution_endpoint}",
+            "collection": "{$.meta.collection}"
```

Comment:
Based on the comments below, I think we might not need `bucket` or `collection` task config for this task

Reply:
```"CumulusMessageAdapterExecutionError: warning setting command to loadAndUpdateRemoteEvent",
    "warning setting command to loadNestedEvent",
    "Unexpected Error <class 'jsonschema.exceptions.ValidationError'>. config schema: 'collection' is a required property",

```
and
```
"errorType": "CumulusMessageAdapterExecutionError",
  "errorMessage": "warning setting command to loadAndUpdateRemoteEvent\nwarning setting command to loadNestedEvent\nUnexpected Error <class 'jsonschema.exceptions.ValidationError'>. config schema: 'collection' is a required property\n\nFailed validating 'required' in schema:\n    {'description': 'Describes the config used by the '\n                    'update-granules-metadata task',\n     'properties': {'bucket': {'description': 'the bucket the has the '\n                                              'private/public key needed '\n                                              'for decrypting cmr password',\n                               'type': 'string'},\n              
```

Reply:
Yeah, so these errors occurred because you removed them from the `task_config`, which sets the `config.collection` value in the task, but you hadn't removed them from the task config schema. So the value of `config.collection` was empty, but the task config schema was still requiring `config.collection` to be present.

So if you remove `bucket` and `collection` from tasks/update-granules-metadata/schemas/config.json and then also from this `task_config`, I would expect those errors to go away.

Reply:
I removed them from both `task_config` in this file and `schemas/config.json` and still ran into the error, but I realized it's because I didn't remove `bucket` and `collections` from the `required` list in `config.json`... oops 😬

Reply:
Ah, makes sense. Glad you figured it out!